### PR TITLE
Milestone 1: Add books & Remove books 

### PR DESCRIPTION
**Pull request by mdg0410**
In this milestone:

- Add logic at the projects
- Add `addBooks` & `removeBook` in the index.html file
- Display books in the index.html file

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+const booksContainer = document.getElementById('containerBooks');
+
+function saveData() {
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function removeBook(index) {
+  books.splice(index, 1);
+  saveData();
+}
```
Comment (AmaduKamara) [inline js/main.js]: [ Optional Changes ]

- [ ] You could have used the array filter method so you remove books by their IDs.

File: js/main.js [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  render();
+};
+
+function addBook(title, author) {
+  books.push({ title, author });
+  render();
+}
```
Comment (AmaduKamara) [inline js/main.js]: [ Optional Changes ]

- [ ] You could have generated an ID or used the author's name as it's unique for a book when it's been added so a book has an ID and is deleted by its ID 👍🏽

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved  🟢 🟢 🟢

Hi Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

##  Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Add and remove book methods work fine 👍🏽
✅ Used an array of objects for storing books 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: js/main.js (hunk 2 above)
Comment (mdg0410) [reply js/main.js]: > Correction

Add ID at the book

File: js/main.js (hunk 1 above)
Comment (mdg0410) [reply js/main.js]: > Correction

Now the books is delete with the method `filter`

**Pull request merged**